_includes/announcement-blog.html

```diff
@@ -0,0 +1,64 @@
+{% assign cohort = include.cohort | strip %}
+{% assign people = site.data.people %}
+{% assign cohort_data = site.data.cohorts[cohort] %}
+{% assign projects = cohort_data.projects %}
+{% assign experts = cohort_data.metadata.experts %}
+
+{% assign cohorts = site.data.cohorts | sort %}
+{% assign current_cohort_index = cohorts | map: 0 | index: cohort %}
+{% assign prev_cohort = cohorts[current_cohort_index | minus: 1][0] %}
+
+<!-- extract previous participants and mentors and count them later among mentors -->
+{% assign prev_cohort = 'ols-' | append: cohort_data.metadata.previous-cohort %}
+{% assign prev_projects = site.data.cohorts[prev_cohort].projects %}
+{% assign prev_participants = '' %}
+{% assign prev_mentors = '' %}
+{% for project in prev_projects %}
+    {% for p in project.participants %}
+        {% capture prev_participants %}{{ prev_participants }}, {{ p }}{% endcapture %}
+    {% endfor %}
+    {% for m in project.mentors %}
+        {% capture prev_mentors %}{{ prev_mentors }}, {{ m }}{% endcapture %}
+    {% endfor %}
+{% endfor %}
+
+{% assign all-participants = '' %}
+{% assign all-p-countries = '' %}
+{% assign all-mentors = '' %}
+{% assign all-m-countries = '' %}
+{% assign all-keywords = '' %}
+{% assign prev-part-count = 0 %}
+{% assign prev-mentor-count = 0 %}
+
+<!-- parse project participants, mentors and their countries -->
+{% for project in projects %}
+    {% for p in project.participants %}
+        {% capture all-participants %}{{ all-participants }}, [{{ people[p].first-name }} {{ people[p].last-name }}](/{{ cohort }}/projects-participants#{{ p }}){% endcapture %}
+        {% capture all-p-countries %}{{ all-p-countries }}, {{ people[p].country }}{% endcapture %}
+    {% endfor %}
+
+    {% for m in project.mentors %}
+        {% capture all-mentors %}{{ all-mentors }}, [{{ people[m].first-name }} {{ people[m].last-name }}](/{{ cohort }}#{{ m }}){% endcapture %}
+        {% capture all-m-countries %}{{ all-m-countries }}, {{ people[m].country }}{% endcapture %}
+<!-- add +1 if participant in previous cohort -->
+        {% if prev_participants contains m %}
+            {% assign prev-part-count = prev-part-count | plus: 1 %}
+        {% endif %}
+<!-- add +1 if mentor in previous cohort -->
+        {% if prev_mentors contains m %}
+            {% assign prev-mentor-count = prev-mentor-count | plus: 1 %}
+        {% endif %}
+    {% endfor %}
+
+<!-- parse keywords -->
+    {% for k in project.keywords %}
+        {% capture all-keywords %}{{ all-keywords }}, {{ k }}{% endcapture %}
+    {% endfor %}
+{% endfor %}
+
+<!-- transform into lists -->
+{% assign p-participants = all-participants | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign p-countries = all-p-countries | remove_first: ', ' | split: ", " | uniq | sort %}
```

Comment:
There is something strange with the countries (extra  ` ,` after `(`): 

![image](https://github.com/open-life-science/open-life-science.github.io/assets/1842467/7d464238-3a4b-4e17-b97e-21d42f562749)

Reply:
This should be fixed now. :)